### PR TITLE
Add more test coverage for functions

### PR DESCRIPTION
**Pull request by cbowman0**
Tests for timeStack, useSeriesAbove, fallbackSeries, mostDeviant and stdev.

### CODE REVIEW

File: webapp/tests/test_functions.py [hunk 1]
```diff
@@ -3457,7 +3686,6 @@ def mock_data_fetcher(reqCtx, path_expression):
 
 
         with patch('graphite.render.evaluator.fetchData', mock_data_fetcher):
-            # input values will be ignored and replaced by regression function
```
Comment (deniszh) [inline webapp/tests/test_functions.py]: Why removing this and comments below? Just curious

Comment (DanCech) [reply webapp/tests/test_functions.py]: Same question, looks great otherwise

Comment (cbowman0) [reply webapp/tests/test_functions.py]: They were erroneously copied from other checks when I created them this week.  I'll see if I can put comments there that do make sense.

**Comment by cbowman0** [comment]
Some comments added back.

Also, unless I missed something, added tests for the final 2 functions with no coverage.

**Pull request merged**